cli/deployment/config.go

```diff
@@ -446,10 +446,19 @@ func newConfig() *codersdk.DeploymentConfig {
 				Default:     512,
 			},
 		},
+		// DEPRECATED: use Experiments instead.
 		Experimental: &codersdk.DeploymentConfigField[bool]{
-			Name:  "Experimental",
-			Usage: "Enable experimental features. Experimental features are not ready for production.",
-			Flag:  "experimental",
+			Name:    "Experimental",
+			Usage:   "Enable experimental features. Experimental features are not ready for production.",
+			Flag:    "experimental",
+			Default: false,
+			Hidden:  true,
+		},
```

Comment:
Should print a deprecation notice at the top of server.go. Or we could implement it as a "Deprecated string" on DeploymentConfigField

Reply:
It's hidden. So it should not matter right?

Reply:
Well to notify old deployments that still have the variable set that they need to unset it or it won't work soon

Reply:
I mean printing a notice if the flag or env var is present

Reply:
warning gets printed in `initExperiments` at the bottom of `coderd.go`; folks may or may not be looking for it though